GraphTest.java

```diff
@@ -0,0 +1,57 @@
+import java.util.HashSet;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * Created by basescu on 06.06.17.
+ */
+
+
+class Dictionary {
+    private static HashSet words;
+    private static HashSet prefixes;
+
+    static {
+        words = new HashSet();
+        prefixes = new HashSet();
+        words.add("CAR");
+        words.add("CARD");
+        words.add("CAT");
+        words.add("CART");
+        prefixes.add("C");
+        prefixes.add("CA");
+        prefixes.add("CAR");
+        prefixes.add("CARD");
+        prefixes.add("CART");
+        prefixes.add("CAT");
+    }
+
+    static boolean isWord(String str) {
+        return words.contains(str);
+    }
+
+    static boolean isPrefix(String str) {
+        return prefixes.contains(str);
+    }
+
+}
+
+class GraphTest {
```

Comment:
No tests for edge cases like empty grid, grid with no words present in the dictionary etc...